pages/booking/list.tsx

```diff
@@ -1,4 +1,4 @@
-import Booking from '../../components/booking'
+import Booking from '../../components/booking/card'
 import GetMorePageData from '../../components/get_more_page_data'
```

Comment:
if this is a function, it shouldn't be in components and also, it shouldn't start with an uppercase letter

Reply:
right. a big mistake